fadtk/fad_calculation.py

```diff
@@ -0,0 +1,21 @@
+from argparse import ArgumentParser
+
+from .fad import FrechetAudioDistance, log
+from .model_loader import *
```

Comment:
let's avoid using `import *`. seems like it was already used in the original repo but no need to make the same mistake on our end.